packages/miniflare/src/workers/core/proxy.worker.ts

```diff
@@ -119,7 +125,11 @@ export class ProxyServer implements DurableObject {
 			// should only ever return `Object`, as none override `Symbol.toStringTag`
 			// https://tc39.es/ecma262/multipage/fundamental-objects.html#sec-object.prototype.tostring
 			const type = getType(value);
-			if ((type === "Object" && !isPlainObject(value)) || type === "Promise") {
+			const isInternalClass = isInternal(value);
```

Comment:
NIT - you probably did this while debugging but there was no real need to assign this to a variable.

Reply:
[remove intermediate variable](https://github.com/cloudflare/workers-sdk/pull/6826/commits/4c65020554c49c431cfc53079465903fadf9e7be)